plugins/com.google.cloud.tools.eclipse.appengine.libraries/src/com/google/cloud/tools/eclipse/appengine/libraries/model/LibraryFile.java

```diff
@@ -78,4 +78,30 @@ public boolean isExport() {
   void setExport(boolean export) {
     this.export = export;
   }
+  
+  @Override
+  public String toString() {
+    return this.mavenCoordinates.toStringCoordinates();
```

Comment:
`this` not needed in these methods.

Reply:
It may be safer to have `toStringCoordinates()` in this class (maybe then with a name `getMavenCoordinatesString()`), because changes in the `MavenCooridnates` class may affect the results of these methods? Not a big deal though.

Reply:
this removed